test/e2e/es_cluster_logging.go

```diff
@@ -75,7 +75,7 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 
 	// Wait for the Elasticsearch pods to enter the running state.
 	By("Checking to make sure the Elasticsearch pods are running")
-	label := labels.SelectorFromSet(labels.Set(map[string]string{"name": "elasticsearch-logging"}))
+	label := labels.SelectorFromSet(labels.Set(map[string]string{"k8s-app": "elasticsearch-logging"}))
```

Comment:
Please make this a constant, rather than a cut 'n pasted string, which is the source of the bug that you're trying to fix here.